### DISCUSSION TITLE
Sanitize Molecule without AdjustHs

### DISCUSSION THREAD

**Discussion by MherMatevosyan** [rdkit/rdkit#3901] [Q&A] [answered]
Hi,

Is it possible to sanitize molecule and tell rdkit to skip a step (AdjustHs)? Currently I just provide all other steps `Chem.SanitizeFlags.SANITIZE_CLEANUP|Chem.SanitizeFlags.SANITIZE_CLEANUPCHIRALITY|Chem.SanitizeFlags.SANITIZE_FINDRADICALS|Chem.SanitizeFlags.SANITIZE_KEKULIZE|Chem.SanitizeFlags.SANITIZE_PROPERTIES|Chem.SanitizeFlags.SANITIZE_SETAROMATICITY|Chem.SanitizeFlags.SANITIZE_SETCONJUGATION|Chem.SanitizeFlags.SANITIZE_SETHYBRIDIZATION|Chem.SanitizeFlags.SANITIZE_SYMMRINGS`. And this just looks cumbersome.

**Comment by ptosco** [comment, marked as answer]
You can use `Chem.SanitizeFlags.SANITIZE_ALL ^ Chem.SanitizeFlags.SANITIZE_ADJUSTHS`.

**Reply by MherMatevosyan** [reply]
It works. Thank you!
